### PR TITLE
Create PRS14127Yohannes.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Scribe of Eth 616

### CODE REVIEW

File: new/PRS14127Yohannes.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="literary"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS14127Yohannes" passive="BNFet616"/>
+                    <relation name="iha:Place_of_activity" active="PRS14127Yohannes" passive="LOC4272Ledata"/>
```
Comment (thea-m) [inline new/PRS14127Yohannes.xml:60]: I think we should not use iha relations?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS14127Yohannes.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS14127Yohannes.xml]: "saws:isRelatedTo" to show that he was affiliated to the Ledata church?

Comment (thea-m) [reply new/PRS14127Yohannes.xml]: syriaca:has-relation-to-place ?

File: new/PRS14127Yohannes.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዮሐንስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Yoḥannǝs</persName>
+                    <persName type="alt"><roleName type="title">ʾalaqā</roleName> Yoḥannǝs</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="literary"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
```
Comment (eu-genia) [inline new/PRS14127Yohannes.xml:56]: add at least 20th century in floruit

Comment (eu-genia) [reply new/PRS14127Yohannes.xml:56]: I actually would also add the activity in Ledata in `floruit` simply with `placeName`, this is what is usually done for persons and places of activity I guess

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14127Yohannes.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS14127Yohannes.xml]: ok

File: new/PRS14127Yohannes.xml (hunk 2 above)
Comment (eu-genia) [reply new/PRS14127Yohannes.xml:56]: I cannot see the corrections, please check

Comment (eu-genia) [inline new/PRS14127Yohannes.xml:56]: still no dates

File: new/PRS14127Yohannes.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS14127Yohannes" passive="BNFet616"/>
+                    <relation name="saws:isRelatedTo" active="PRS14127Yohannes" passive="LOC4272Ledata"/>
```
Comment (eu-genia) [inline new/PRS14127Yohannes.xml:60]: syriaca relation (`saws `is only for works)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14127Yohannes.xml (hunk 3 above)
Comment (eu-genia) [reply new/PRS14127Yohannes.xml:60]: can these be revised? @DenisNosnitsin1970

**Review by eu-genia — APPROVED** [review]

**Pull request merged**